multi-agent/lib/mas/blueprints/exceptions.py

```
@@ -41,3 +41,10 @@ def __init__(self, blueprint_id: str, message: str = None):
         self.message = message or f"Failed to update metadata for blueprint '{blueprint_id}'"
         super().__init__(self.message)
 
+
+class PromptShortcutsValidationError(BlueprintError):
+    """Raised when prompt shortcuts fail validation (bad text, title too long, >3 items)."""
+    def __init__(self, message: str):
```

Ruff (0.15.18)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)